threema/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: threema
+name: Threema Web
+tagline: An intuitive and secure web client for Threema users
+category: files
+version: "2.6.1"
+port: 88
```

Comment:
Please use a port higher than 1023. Ports below that are called system ports and are reserved for other purposes. See: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Well-known_ports